### PR TITLE
ci(tests): add mini dataset to stateful test

### PR DESCRIPTION
**Pull request by ZeaLoVe**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Add mini dataset to stateful test, include ontime and hits

Fixes #issue

### CODE REVIEW

**Comment by Xuanwo** [comment]
IMHO, this test should be added as a separate github action because we can't run them locally.

File: tests/suites/1_stateful/04_mini_dataset/00_0001_mini_hits.sh
Comment (Xuanwo) [inline tests/suites/1_stateful/04_mini_dataset/00_0001_mini_hits.sh:14]: Write as array so that we don't need to write `echo xxx | xxx`.

For example:

https://github.com/datafuselabs/databend/blob/3e0ec788e14373b0925dac26475536159c272094/tests/suites/1_stateful/00_copy_v2/00_0000_copy_from_s3_location.sh#L11-L34

**Comment by Xuanwo** [comment]
I fixed the permission denied problem, hoping this will help you.

File: tests/suites/1_stateful/04_mini_dataset/00_0000_mini_ontime.sh
Comment (Xuanwo) [inline tests/suites/1_stateful/04_mini_dataset/00_0000_mini_ontime.sh:11]: Please enable the new planner for all queries.

**Review by Xuanwo — CHANGES_REQUESTED** [review]

**Comment by Xuanwo** [comment]
`repo.databend.rs` is a public-read bucket, I think we don't need to set ak/sk here.

Should work after https://github.com/datafuselabs/databend/pull/6623

**Comment by Xuanwo** [comment]
@mergify update

**Comment by Xuanwo** [comment]
This PR is blocked by https://github.com/datafuselabs/databend/pull/6624

**Comment by Xuanwo** [comment]
@mergify update

**Comment by Xuanwo** [comment]
I'm fighting with GitHub's actions.

**Comment by Xuanwo** [comment]
After https://github.com/datafuselabs/opendal/issues/442, we can download data from CDN instead.

As in this PR, let's fall back to `curl` and COPY from the internal stage.

**Comment by Xuanwo** [comment]
Blocked by https://github.com/datafuselabs/databend/pull/6691

**Comment by Xuanwo** [comment]
@ZeaLoVe The test data can load now. Can you help fix the test results?

**Comment by ZeaLoVe** [comment]
> @ZeaLoVe The test data can load now. Can you help fix the test results?

OK, I will turn to this tomorrow.

**Comment by Xuanwo** [comment]
@mergify update

**Comment by ZeaLoVe** [comment]
cc @sundy-li   seems hits.sql could not create table success, I run it local, see this error.

WlJaM8tcjd

set enable_planner_v2 = 1;    still error.

**Comment by ZeaLoVe** [comment]
Continue after these PRs merged. #6880 #6899

**Comment by Xuanwo** [comment]
@mergify update

**Comment by Xuanwo** [comment]
> Continue after these PRs merged. #6880 #6899

PTAL

**Comment by ZeaLoVe** [comment]
@mergify update

**Comment by ZeaLoVe** [comment]
Maybe split this pr is a good choice, one for ontime, the other for hits. Then we can track problem clearly. Now the hits always timeout , any idea? cc @Xuanwo

**Comment by Xuanwo** [comment]
> Maybe split this pr is a good choice, one for ontime, the other for hits. Then we can track problem clearly. Now the hits always timeout , any idea? cc @Xuanwo

LGTM.

Let's merge the test case for ontime first.

**Comment by ZeaLoVe** [comment]
Already split into 2 pr, close here.

**Pull request closed without merging**
